source/features/follow-file-renames.tsx

```diff
@@ -8,51 +8,66 @@ import * as api from '../github-helpers/api';
 import GitHubURL from '../github-helpers/github-url';
 
 interface File {
-	previous_filename: string;
+	previous_filename: string | undefined;
 	filename: string;
 	status: string;
+	blob_url: string;
 }
-// eslint-disable-next-line import/prefer-default-export
-export async function getCommitInfo(oid: string): Promise<AnyObject> {
+export interface FileChanges {
+	file: File;
+	commit: {
+		parentSha: string | undefined;
```

Comment:
When would this undefined?

Reply:
Maybe if it is the first commit? But then it won't be renamed or deleted - I'll change that